frontend/src/scenes/surveys/Survey.tsx

```diff
@@ -42,6 +42,14 @@ export const scene: SceneExport = {
     }),
 }
 
+export const SurveyQuestionLabel = {
+    [SurveyQuestionType.Open]: 'Open text',
+    [SurveyQuestionType.Rating]: 'Rating',
+    [SurveyQuestionType.Link]: 'Link',
+    [SurveyQuestionType.SingleChoice]: 'Single choice select',
+    [SurveyQuestionType.MultipleChoice]: 'Multiple choice select',
+}
```

Comment:
Perhaps this should live in _constants.tsx_?

Reply:
yep, or in a separate surveys constants/util file since there's a lot of defaults being defined in `surveyLogic` that can be moved out